client/src/state/actions/saveAction.js

```diff
@@ -0,0 +1,16 @@
+//Saving what user types in input as payload. Will come back to this later
+// export function saveExercise(exerciseData){
+//     return {
+//         type: 'SAVE_EXERCISE',
+//         payload: exerciseData.target.value,
+//         key: exerciseData.target.name
+//     }
+// }
+
+export function saveExercise(exerciseData){
+    return {
+        type: 'SAVE_EXERCISE',
+        payload: exerciseData.target.value,
+        key: exerciseData.target.name
```

Comment:
it could save you from problems down the road if your action creator just takes plain `name` and `value` parameters, instead of relying on the shape of an event object; for example if you had to save an exercise from somewhere other than an event handler, you'd have to write another action creator that does the same thing.